chibuezeayogu/ecommerce-api
# Implement product search

**What does the PR do?**
- This PR implements `Product Search` endpoint

**Description of Task to be completed?**
- Ensures users can search for a product

**How should this be manually tested?**
- Clone the repository [Ecommerce-API](https://github.com/chibuezeayogu/Ecommerce-API.git) and
-  cd into `Ecommerce-API` and checkout to this branch `ft-implement-product-search`
- run `bundle install` to install new `gems`
- run `rails server` and test these routes using `Postman`
           ` GET /products/search` 
- run `rspec` to ensure all tests are passing
